sdk/cognitiveservices/ms-azure-cs-textanalytics/README.md

```diff
@@ -0,0 +1,22 @@
+## Microsoft Azure SDK for Python
+
+**NOTE:** If you are looking for the new, preview version of Azure Text
```

Comment:
new, ~~preview~~ version

Reply:
I'd be even more explicit here. Something like "This is a legacy version that works with the xxx version of the service API. For the new set of libraries that support newer service versions please see [azure-ai-textanalytics](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/textanalytics/azure-ai-textanalytics)"

Reply:
It will be good to check with Assaf too just in case

Reply:
@assafi what I need to do for the Track 1 Text Analytics SDK which is based on V2.0 API?

Reply:
@mssfang 
Here is the Track 1 Python version - https://pypi.org/project/azure-cognitiveservices-language-textanalytics
It works against the v2.1 API. 
More examples here - https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/quickstarts/text-analytics-sdk?tabs=version-2&pivots=programming-language-python

Reply:
Thank you. Assaf.